site/content/blog/simplifying-package-authoring.md

```diff
@@ -0,0 +1,301 @@
+```
+
+As we can see that `dynatrace-operator` pod is up, it means the package is behaving as expected. Now, we will use `kctrl pkg release` to create the package which will be released.
+
+### kctrl pkg release
+
+```bash
+$ kctrl pkg release -v 1.0.0
+```
+
+This command will create an imgpkg bundle, upload it to the OCI registry and create `package.yml` and `metadata.yml` which can be released for consumption eventually. In this command, it will ask only 1 question about where to push our imgpkg bundle. Also, we are versioning our package with `-v` flag.
+
+![Simplifying package authoring - pkg release basic details](/images/blog/simplifying-package-authoring-package-release.png)
+
+While entering the registry URL, ensure to change the value from `docker.io/rohitagg2020/dynatrace-bundle` to `docker.io/<YOUR_DOCKERHUB_ACCOUNT>/dynatrace-bundle`. Alternatively, you can enter other valid OCI registry URL.
+
+Now, we have created our package and packageMetadata. 
+
+Let's see how the package and packageMetadata files look like:
+
+```bash
+$ cat carvel-artifacts/packages/dynatrace.carvel.dev/package.yml
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: dynatrace.carvel.dev.1.0.0
+spec:
+  refName: dynatrace.carvel.dev
+  releasedAt: "2022-08-15T11:54:50Z"
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - imgpkgBundle:
+          image: index.docker.io/rohitagg2020/dynatrace-bundle@sha256:d3fd67881ccba75134061451348130577ad0b2034d0a65b44b965b56e7d1c939
+      template:
+      - ytt:
+          paths:
+          - upstream
+      - kbld:
+          paths:
+          - '-'
+          - .imgpkg/images.yml
+  valuesSchema:
+    openAPIv3:
+      default: null
+      nullable: true
+  version: 1.0.0
+
+$ cat carvel-artifacts/packages/dynatrace.carvel.dev/metadata.yml
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: dynatrace.carvel.dev
+spec:
+  displayName: dynatrace
+  longDescription: dynatrace.carvel.dev
+  shortDescription: dynatrace.carvel.dev
+```
+
+Next step is to add it to package repository. 
+
+### kctrl pkg repo release
+
+`kctrl` can be used to release packages grouped together as a `PackageRepository`. Let’s bundle the `dynatrace` package created above into the PackageRepository.
+
+Let's create a folder for our repository.
+```bash
+$ cd ../
+$ mkdir pkg-repo && cd pkg-repo
+$ mkdir packages && cd packages
+```
+
+`kctrl` will create a repo bundle from all the packages and metadata present in the `packages` folder. 
+We will copy the `package.yml` and `metadata.yml` created above into the `packages` folder. We will follow the bundle format as mentioned [here](). Alternatively, while running `pkg release`, `--repo-output` flag can be used to copy the `package.yml` and `metadata.yml` in the prescribed PackageRepository bundle format at a specified location.
```

Comment:
make sure to add a link to the [here]() as mentioned

Reply:
Will add it once the Doc PR is merged.